tests/migration/online_migration/yast2_migration.pm

```diff
@@ -218,7 +218,14 @@ sub run {
         # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
         if (get_var('SMT_URL') =~ /smt/) {
             assert_screen 'import-untrusted-gpg-key', 180;
-            send_key 'alt-t';
+            # for 15-SP7 yast2 migration with smt, we need to choose Again and Yes.
+            if (check_var('VERSION', '15-SP7')) {
+                send_key 'alt-a';
```

Comment:
Why need send alt-a?

Reply:
https://openqa.suse.de/tests/16124624#step/yast2_migration/7 for `Do not show this message Again`.

Reply:
Aha, yes.